src/Core/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -108,8 +108,11 @@ public static void AddSqlServerRepositories(this IServiceCollection services, Gl
             }
         }
 
-        public static void AddBaseServices(this IServiceCollection services)
+        public static void AddBaseServices(this IServiceCollection services, GlobalSettings globalSettings)
```

Comment:
Why in `AddBaseServices` vs. `AddDefaultServices` which already relies on `GlobalSettings`? Is that not called from each of the associated `Startup.cs` routines (seems like it is)? Just curious if the order matters.

Reply:
@cscharf I considered adding it to `AddDefaultServices`, but my thinking was it would be possible to just call `AddBaseServices` without calling `AddDefaultServices`. But I see now that the dependencies in `AddBaseServices` relies on `AddDefaultServices` so we would have issues either way.